packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -111,18 +144,35 @@ abstract class FlutterCommand extends Command<Null> {
   /// and [runCommand] to execute the command
   /// so that this method can record and report the overall time to analytics.
   @override
-  Future<Null> run() {
-    final Stopwatch stopwatch = new Stopwatch()..start();
-    final UsageTimer analyticsTimer = usagePath == null ? null : flutterUsage.startTimer(name);
+  Future<Null> run() async {
+    final DateTime startTime = clock.now();
```

Comment:
By starting this here, we'll record the time it takes to update the cache if needed (which is done in `verifyThenRunCommand()`), which will mess with the analytics results...